gridfinity-rebuilt-baseplate.scad

```diff
@@ -87,7 +90,7 @@ gridfinityBaseplate([gridx, gridy], l_grid, [distancex, distancey], style_plate,
  * @param sh Style of screw hole allowing the baseplate to be mounted to something.
  * @param fit_offset Determines where padding is added.
  */
-module gridfinityBaseplate(grid_size_bases, length, min_size_mm, sp, hole_options, sh, fit_offset = [0, 0]) {
+module gridfinityBaseplate(grid_size_bases, length, min_size_mm, sp, hole_options, sh, st, fit_offset = [0, 0]) {
```

Comment:
I'd prefer a more descriptive variable name.  `st` requires me to look at other locations to figure out what it's doing.